### PR TITLE
Feature/DTGB-559: Added theming for outdoor location detail pages.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: templates/custom/nodes/node--terrain--default.html.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{% endif %}
+
+{% if content.field_images|render|striptags|trim is not empty %}
+  <div class="image-gallery-wrapper multiple">
+    {{ content.field_images }}
+  </div>
+{% endif %}
+
+{% if content.field_playground_offer|render|striptags|trim is not empty %}
+  <h3>{{ 'Park offer'|t }}</h3>
```
Comment (MPParsley) [inline templates/custom/nodes/node--terrain--default.html.twig:25]: klopt deze string? in het design staat "Ik wil buiten spelen of sporten" bij de facet.

File: templates/custom/nodes/node--terrain--default.html.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<h2>{{ label }}</h2>
+
+{% if content.field_park_offer|render|striptags|trim is not empty %}
+  <h3>{{ 'Park offer'|t }}</h3>
```
Comment (MPParsley) [inline templates/custom/nodes/node--terrain--default.html.twig:4]: klopt deze string? in het design staat "Ik wil naar een park" bij de facet.

**Review by MPParsley — APPROVED** [review]

File: templates/custom/nodes/node--terrain--default.html.twig (hunk 1 above)
Comment (meirege) [reply templates/custom/nodes/node--terrain--default.html.twig:25]: Das niet die string. Klopt wel denk ik. :)

File: templates/custom/nodes/node--terrain--default.html.twig (hunk 2 above)
Comment (meirege) [reply templates/custom/nodes/node--terrain--default.html.twig:4]: Das niet die string. Klopt wel denk ik. :)

**Pull request merged**